postinst

```diff
@@ -0,0 +1,2 @@
+mkdir -p /opt/cs50/bin/
+ln -s /opt/cs50/phpliteadmin/bin/phpliteadmin -t opt/cs50/bin/
```

Comment:
Oops, typo. Fixed and forced symlinking to avoid error if symlink is already there for some reason a10bf2c.